iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/TokenRenewalAndroidRunner.java

```diff
@@ -20,13 +20,4 @@
 {
     @Rule
     public Rerun count = new Rerun(3);
```

Comment:
This might be a good time to remove these test re-runs

Reply:
Sure thing